package.json

```diff
@@ -36,7 +36,7 @@
     "clean:root": "rimraf node_modules",
     "build": "lerna run build --sort --concurrency=1 --stream",
     "build:quick": "yarn build -- --scope=@interledger/*",
-    "test": "lerna run test --concurrency=1 --stream",
+    "test": "lerna run test --sort --concurrency=1 --stream",
```

Comment:
I don't think this is needed, and --sort is apparently lexical not topological ... 
some other commands use --toposort

Reply:
I think the default is --toposort